.travis.yml

```diff
@@ -53,7 +53,7 @@ matrix:
           android:
             components:
               - build-tools-25.0.2
-              #Including script to run on power
+ #Including script to run on power
```

Comment:
You add the new test targets in the middle of the Android NDK (CMake). I think Line 49 is a good way to add new ones.